doc/releases/release-notes-4.1.rst

```diff
@@ -93,6 +93,8 @@ Boards & SoC Support
 
 * Made these board changes:
 
+  * Enabled RTC, PIT on NXP ``frdm_mcxc444``
```

Comment:
@NeilChen93 , apologies for the back and forth on this. Upon further discussion with the 4.1 release managers it was decided that we do not want to include such changes in the release notes. Kindly delete this addition.

Reply:
I dropped the commit for you @NeilChen93

Reply:
Thanks